pkg/virtctl/usbredir/usbredir_s390x.go

```diff
@@ -0,0 +1,34 @@
+//go:build s390x
+
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package usbredir
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type usbredirCommand struct {
+	clientConfig clientcmd.ClientConfig
```

Comment:
nit: you don't need this field

Reply:
Good catch, moved it back into the original, shared file.